user/enterprise/tcie-3.x-obtain-license.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Obtaining Travis CI 3.x license
+layout: en_enterprise
+
+---
+
+In order to obtain your Travis CI Enterprise 3.x license, please follow instructions below.
+
+## New customers
+
+Please refer to general [Travis CI Enterprise information](/user/enterprise/#trials-and-licensing) for instructions.
+
+## Existing customers
+
+1. Contact Travis CI support team providing:
+    1. Your customer email (the one in currently active license)
+    2. A hostname, where the deployment will be done (using wildcard format, *.yourdomain.here)
+    3. Will you be using GitOps to manage your k8s cluster? Y/N
+    4. Will your installation be air gapped? Y/N
+    <br />The number of seats and other license parameters will be transferred from your current active license.
+2. Our Support Team will issue to you a Trial License for 60 days in order to enable your tests and database migrations.
+3. After your tests are completed and you have an environment ready to go, please contact Travis CI Support, requesting a production 4. license for Enterprise 3.0 and your existing 2.0 license will be invalidated within 24 hours.
```

Comment:
For existing customers, do we want to consider keeping their 2.0 license for a little longer than 24hrs to confirm their TCIE 3.0 setup is up and running smoothly? If we are open to this, I suggest 1-week.

Reply:
@muizzk adressed with subsequent commit as per internal discussion